PHYS_434_Lab1.html

```diff
@@ -0,0 +1,176 @@
jqVUC6bFLAVBQEAAGDdTpw4oatXr+rOnTtqyufzun37tvr7+/XQ+Pi4RkZGVK/X1XTixAldvXpVd+7cUVM+n9ft27fV39+vh06cOKGrV6/qzp07asrn87p9+7b6+/vV1N3drS9/+cv6wQ9+oE8++URNP//5z3XlyhUNDg4qHo9rvaYWqoqSTiYEbFVBAAAAWLfh4WEdPnxYfX196uzs1OnTp3XmzBn19vbqobm5OX344YdqNBpqGh4e1uHDh9XX16fOzk6dPn1aZ86cUW9vrx4aHh7W4cOH1dfXp87OTp0+fVpnzpxRb2+vHvrRj36kVCql1157TZ2dnXrjjTf0zW9+U9/5znf0NAqlFUXJZZMCtqogAAAArFsIQRcuXJC7q1gsqlgsanBwUKu9/fbbcndt27ZNTSEEXbhwQe6uYrGoYrGowcFBrRZC0IULF+TuKhaLKhaLGhwc1GodHR364IMP5O4qFov67W9/q5GRET2NcqWmQqmqKIf27RKwVQUBAAAAf1NeeaAo6VRCuWxSwFYVBAAAAPzN1EJVUdLJhICtLAgAAAD4m0JpRVFy2aSArSwIAAAA+JvySk1RDu3bJWArCwIAAAD+S7lSU7lSU5RcNilgKwsCAAAA/kt55YGipFMJAVtdEAAAAPBfphaqipLLJgVsdUEAAADAfymUVhTl9Z4vCNjqggAAAND2ypWaCqWqgHYXBAAAgLZXXnmgKOlUQrlsUsBWFwQAAIC2N7VQVZR0MiGgHQQBAACg7RVKK4qSyyYFtIMgAAAAtL1Cqaooh/btEtAOggAAANDWypWaoqRTCeWySQHtIAgAAABtrVBaEYC/CgIAAEBbG5tZUpSB7g4B7SIIAAAAiHBo3y4B7SIIAAAAbatcqalQqgrAXwUBAACgbZVXHihKOpVQLpsU0C6CAAAA0LamFqqKkk4mBLSTIAAAALStQmlFUXLZpIB2EgQAAIC2VShVFeXQvl0C2kkQAAAA2lK5UlOUdCqhXDYpoJ0EAQAAoC2VVx4IwD8KAgAAQFsam/mDogx0dwhoN0EAAABoS+XKA0U5tG+XgHYTBAAAgLZTrtRUKFUF4B8FAQAAoO2UVx4oSjqVUC6bFNBuggAAANB2phaqipJOJgS0oyAAAAC0nUJpRVFy2aSAdhQEAACAtlNeqSnKoX27BLSjIAAAALSVcqWmcqWmKLlsUkA7CgIAAEBbKa88UJR0KiGgXQUBAACgrUwtVBUlnUwIaFdBAAAAaCuF0oqi5LJJAe0qCAAAAG2jXKmpUKoqyqF9uwS0qyAAAAC0jfLKA0VJpxLKZZMC2lUQAAAA2sbUQlVR0smEgHYWBAAAgLZRKK0oSi6bFHVVSKwAABI2SURBVNDOggAAAIC/ObRvl4B2FgQAAIC28D/+5fMqlKqKkk7+i4B2FgQAAIC28D/+l88rSjqVUDqVENDOggAAANAWHnz+y4qSTiYEtLsgAAAAtIX7n/9vipLLJgW0uyAAAAC0hQef/2+KcmjfLgHtLggAAABbXrlS01py2aSAdhcEAACALa+88kBR0qmEAEhBAAAA2PKmFqqKkk4mBEAKAgAAwJZXKK0oyus9HQIgBQEAAGBLK1dqKpSqipJOJQRACgIAAMCWVl55oCjpVEK5bFIApCAAAABsaVMLVUVJJxMC8FdBAAAA2NIKpRVFyWWTAvBXQQAAAGhrh/btEoC/CgIAAMCWVa7UVChVFSWd/BcB+KsgAAAAbFnllQeKkk4llE4lBOCvggAAALBlTS1UFSWdTAjA3wUBAABgyyqUVhQll00KwN8FAQAAYMsqr9QU5dC+XQLwd0EAAADY1O7fv698Pq+xsTFNTk7qoXKlpnKlpii5bFIA/i4IAAAAm9by8rKOHz+uy5cva3Z2ViMjIzp58qTq9brKKw8UJZ1KCMA/CgIAAMCmdfHiRe3cuVPj4+O6dOmS8vm8FhYWNDo6qqmFqqKkkwltde+++66AJxEEAACATaler2tiYkKnTp1SPB5X0549e3Ts2DHduHFDhdKKouSySW117733noAnEQQAAIBNaW5uTvV6XXv37tVqBw4c0Pz8vN4+nNa5Ixnlsrv0KP/3+f9TZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMTWYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZ6dVXXxWeThAAAAA2pWq1qqauri6ttmPHDjX9b19M6OzRjD4aPqj/+X/9H1r8z3/TR8MHdO5IRrnsLn0y/f/K3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u95//33h6QQBAABgU2o0GnoS6VRCuWxSZ49m9NHwQQH4Z0EAAADYlGKxmJpqtZpWazQaaorFYgLwZIIAAACwKWUyGTXNz89rteXlZW3fvl3xeFwAnkwQAAAANqV0Oq3du3erWCxqtenpaXV3dwvAkwsCAADApnXixAldvXpVd+7cUVM+n9ft27fV398vAE8uCAAAAJvW8PCwDh8+rL6+PnV2dur06dM6c+aMent7BeDJB
```

Comment:
Make sure you discuss the results of each calculation. Do they make sense to you? Why?